### PR TITLE
Add dependabot for pip, Docker, and Git Submodules

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+updates:
+  - commit-message:
+      prefix: "Changelog:All"
+    directory: /tests/acceptance/
+    package-ecosystem: pip
+    schedule:
+      interval: weekly
+  - commit-message:
+      prefix: "Changelog:All"
+    directory: /meta-mender-qemu/docker/
```
Comment (lluiscampos) [inline .github/dependabot.yml:11]: @mzedel I am unsure about this one, the Dockerfile(s) are at `/meta-mender-qemu/docker/*/Dockerfile`

Comment (mzedel) [reply .github/dependabot.yml:11]: I don't know it for a fact, but I'm pretty sure these would have to be added as individual entries in the dependabot file (so one with _qemux86-64_ and one with _vexpress-qemu_ as the directory)

**Review by mzedel — CHANGES_REQUESTED** [review]
left a comment for the `docker` updates as I think that would fail, the only other nitpick is the trailing slash for the directory, but this should not cause problems...

**Comment by lluiscampos** [comment]
> left a comment for the `docker` updates as I think that would fail, the only other nitpick is the trailing slash for the directory, but this should not cause problems...

Thanks. It should be fixed now :+1:

**Review by mzedel — APPROVED** [review]

**Pull request merged**